packages/pds/src/api/app/bsky/actor/putPreferences.ts

```diff
@@ -19,7 +19,11 @@ export default function (server: Server, ctx: AppContext) {
         }
       }
       await ctx.actorStore.transact(requester, async (actorTxn) => {
-        await actorTxn.pref.putPreferences(checkedPreferences, 'app.bsky')
+        await actorTxn.pref.putPreferences(
+          checkedPreferences,
+          'app.bsky',
+          auth.credentials.scope,
+        )
```

Comment:
Part of me likes "terminating" scopes and access control in the handlers.  That would involve not passing a scope into `putPreferences()`, instead passing a function or perhaps a deny-list based on the scope.

That said, it's super minor and definitely arguable either way, so mostly just making a note for future thought if we run into more cases like this.

Reply:
I actually started out like that, but then switched it to passing the scope through so we could share the filtering logic & didn't have to pass around magic lists of hard coded strings in the handlers 🤔 

I get what you're pointing at, but think I'll just keep it like this for now - we'll get to clean all this up when we get around to the private state api